@AH-64D Apache Longbow/Addons/fza_ah64_systems/headers/systems.hpp

```diff
@@ -0,0 +1,29 @@
+#define SYS_APU_DMG_VAL   0.85
+#define SYS_BATT_DMG_VAL  0.85
+#define SYS_GEN_DMG_VAL   0.85
```

Comment:
What is the intention of `_VAL`, could `_THRESHOLD` be better?

Reply:
Because VAL is shorter to type, THRESH is acceptable I guess...